CHANGES.rst

```diff
@@ -1,7 +1,102 @@
+improved in the stats definition table :ref:`stats_notation`.
+
+Many additional methods to modify the data have been added to the ``MapDataset``.
+This includes ``.downsample()``, ``.pad()``, ``.resample_energy_axis()``, ``.slice_by_energy()``
+``.slice_by_idx()``. The models definition in the dataset is now reset consistently
+in all of the listed methods. The information returned by ``.info_dict()`` has been
+unified. The information contained in ``.meta_table`` is now handled correctly
+during stacking of datasets.
+
+
+Following the proposal in `PIG 21`_ Gammapy v0.18 comes with many improvements
+to the ``gammapy.modeling.models`` sub-package. This includes the introduction
+of "norm" spectral models, which are multiplicative models to used with spatial
+and spectral template or parametric models. Among those are a
+``PowerLawNormSpectralModel``, ``LogParabolaNormSpectralModel`` and
+``PiecewiseNormSectralModel``. The EBL absorption was refactored
+accordingly to an ``EBLAbsorptionNormSpectralModel``. A new
+``GeneralizedGaussianSpatialModel`` has been introduced. Gammapy v0.18 comes
+now with support for custom energy dependent spatial models. The ``SkyDiffuseCube``
+had been removed, the same functionality can now be achieved with the ``TemplateSpatialModel``.
+Following the proposal in `PIG 21`_, short YAML tags were introduced for
+all models. An overview of the tags can be found in a table in the linked
+PIG document.
+
+Gammpy v0.18 now supports an ``EDispKernelMap``, which stores a spatially
+dependent energy redistribution matrix. This allows to handle the safe
+energy threshold for stacked analyses correctly. The ``MapDataset.edisp``
+attribute has been changed to this class by default. The ``IRFStacker`` class has been removed.
+
+All ``Estimator`` classes support now the definition of energy bins on init. This
+means light-curves, excess maps, excess profiles and ts maps can be computed in
+energy bins with the same API.
+
+An ``Fit.stat_surface()`` method was introduce which allows to compute a
+fit statistic surface. In addition am option to store the trace of the fit was
+added. Both are demonstrated in dedicated sections in the `modeling and fitting tutorial <tutorials/modeling.html>`__
+
+Following the proposal in `PIG 19`_, the ``gammapy.time`` sub-package was removed.
+All functionality related to light curves can be found in ``gammapy.estimators``.
+The Feldman-Cousins method has been removed from the ``gammapy.stats``.
+
+Gammapy v0.18 comes with an improved performance related to caching the computation
+of predicted counts and caching of psf convolution. In Gammapy v0.18 the support
+for Numpy<1.17 had been dropped.
+
+
+**Contributors:**
+
+In alphabetical order by first name:
+
+- Atreyee Sinha
+- Axel Donath
+- Fabio Pintore
+- José Enrique Ruiz
+- Luca Giunti
+- Quentin Remy
+- Régis Terrier
+- Laura Olivera Nieto
+- Alexis de Almeida Coutinho
```

Comment:
This violates alphabetic order - Alexis should be first and Laura comes before Luca

Reply:
The list misses:
Bruno Khelifi
Cosimo Nigro
Fabio Acero
Lea Jouvin

Reply:
Also
Jalel Hajlaoui